app/FAIRshakeWeb/view.py

```diff
@@ -43,8 +55,129 @@ def register():
     current_user={},
   )
 
-@app.route('/login', methods=['GET'])
-def login():
-  return render_template('login.html',
+@app.route('/project/<int:project>/resources', methods=['GET'])
+def resources(repository: RepositoryAPI, assessment: AssessmentAPI, score: ScoreAPI, project):
+  resources=repository.get(tag=project)
+  current_user_assessments = []
+  assessment_count = {}
+  aggregate_scores = {}
+  for resource in resources: #resource-id=1-7
+    try:
+      assessment.get(object=resource.id, user=current_user['id']) ### redesign, if there is an assessment for this object for this user
+      current_user_assessments.append(resource.id) # just array with resource ids
+    except:
+      pass
+    assessment_count[resource.id] = len(assessment.get(object=resource.id)) # all assessments from all users for all rubrics
+    aggregate_scores[resource.id] = score.get(id=resource.id,kind='text/html') # scores is an array of html with tooltip script
+  current_user_assessments=['1'] ###
+  return render_template('project_resources.html',
+    project=repository.get(id=project,limit=1), ###
+    resources=resources,
+    aggregate_scores=aggregate_scores,
+    assessment_count=assessment_count,
+    current_user_assessments=current_user_assessments,
+    current_user={},
+  )
+
+@app.route('/project/<int:project>/my_evaluations', methods=['GET'])
+def my_evaluations(repository: RepositoryAPI, assessment: AssessmentAPI, score: ScoreAPI, project):
+  project_resources=repository.get(tag=project)
+  current_user_assessed_resources = []
+  assessment_count = {}
+  aggregate_scores = {}
+  current_user_scores = {}
+  for resource in project_resources:
+    try:
+      assessment.get(object=resource.id, user=current_user['id'])
+      current_user_assessed_resources.append(resource)
```

Comment:
Same as above, with respect to the appending and the try...except statement.

Reply:
```
current_user_assessed_resources = [
    resource for resource in repository.get(tag=project)
    if assessment.get(object=resource.id, user=current_user['id'])
  ]
```
?